import { catchError } from 'rxjs/operators';
/**
 * catchError with error logging operator
 *
 * @param fn - callback function
 */
export function catchErrorWithLog(fn) {
    return (source) => source.pipe(catchError((state) => {
        console.error(state);
        return fn(state);
    }));
}
//# sourceMappingURL=operators.util.js.map